src/Components/ImagesTable/ClonesTable.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useEffect, useState } from 'react';
+
+import { ClipboardCopy, Skeleton } from '@patternfly/react-core';
+import {
+  TableComposable,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from '@patternfly/react-table';
+
+import { StatusClone, AwsDetailsStatus } from './Status';
+
+import {
+  ClonesResponseItem,
+  ComposesResponseItem,
+  UploadStatus,
+  useGetCloneStatusQuery,
+  useGetComposeClonesQuery,
+  useGetComposeStatusQuery,
+} from '../../store/imageBuilderApi';
+
+type RowPropTypes = {
+  ami: JSX.Element;
+  region: JSX.Element;
+  status: JSX.Element;
+};
+
+type AmiPropTypes = {
+  status: UploadStatus | undefined;
+};
+
+const Ami = ({ status }: AmiPropTypes) => {
+  switch (status?.status) {
+    case 'success':
+      return (
+        <ClipboardCopy
+          hoverTip="Copy"
+          clickTip="Copied"
+          variant="inline-compact"
+        >
+          {'ami' in status.options ? status.options.ami : null}
+        </ClipboardCopy>
+      );
+
+    case 'failure':
+      return undefined;
+
+    default:
+      return <Skeleton width="12rem" />;
+  }
+};
+
+const ComposeRegion = () => {
+  return <p>us-east-1</p>;
+};
+
+type CloneRegionPropTypes = {
+  region: string;
+};
+
+const CloneRegion = ({ region }: CloneRegionPropTypes) => {
+  return <p>{region}</p>;
+};
+
+const Row = ({ ami, region, status }: RowPropTypes) => {
+  return (
+    <Tbody>
+      <Tr className="no-bottom-border">
+        <Td dataLabel="AMI">{ami}</Td>
+        <Td dataLabel="Region">{region}</Td>
+        <Td dataLabel="Status">{status}</Td>
+      </Tr>
+    </Tbody>
+  );
+};
+
+type CloneRowPropTypes = {
+  clone: ClonesResponseItem;
+};
+
+const CloneRow = ({ clone }: CloneRowPropTypes) => {
+  const [pollingInterval, setPollingInterval] = useState(8000);
+
+  const { data: status } = useGetCloneStatusQuery(
+    {
+      id: clone.id,
+    },
+    { pollingInterval: pollingInterval }
+  );
+
+  useEffect(() => {
+    if (status?.status === 'success' || status?.status === 'failure') {
+      setPollingInterval(0);
+    } else {
+      setPollingInterval(8000);
+    }
+  }, [setPollingInterval, status]);
+
+  return (
+    <Row
+      ami={<Ami status={status} />}
+      region={<CloneRegion region={clone.request.region} />}
+      status={<StatusClone clone={clone} status={status} />}
+    />
+  );
+};
+
+type ComposeRowPropTypes = {
+  compose: ComposesResponseItem;
+};
+
+const ComposeRow = ({ compose }: ComposeRowPropTypes) => {
```

Comment:
As there's no polling, it seems that the ami of parent compose gets stuck in the skeleton phase on failure. :skull: 

![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/9f120002-e77d-4683-89b4-1b7966ec9c25)